tests/models_test.py

```diff
@@ -191,11 +196,15 @@ def test_seq2seq_train(self):
     output_seq_len = 5
     input_vocab_size = 10
     output_vocab_size = 20
-    cfg = get_default_config()
+    config: CfgNode
+    config = get_default_config();
```

Comment:
Feels redundant to declare it and then assign its value to the default config, what was wrong with the previous version? Couldn't you merge the 'opts' to the cfg?